server/src/internalClusterTest/java/org/opensearch/indices/stats/IndexStatsIT.java

```diff
@@ -1418,6 +1422,42 @@ public void testConcurrentIndexingAndStatsRequests() throws BrokenBarrierExcepti
         assertThat(executionFailures.get(), emptyCollectionOf(Exception.class));
     }
 
+    public void testZeroRemoteStoreStatsOnNonRemoteStoreIndex() {
```

Comment:
The IndexStatsIT suite creates non-remote backed indices. This test checks that for nodes hosting non-remote store backed indices, the corresponding remote-store stats are 0

Reply:
Got it. Just to confirm: We will omit these fields for non remote store index in the API response, right?

Reply:
We are not omitting these fields here just to keep the nodes stats API output pattern consistent. But we will be showing 0 values for these fields, just like how other fields in this API works. For instance, if there are no indices in the cluster, the nodes stats API would show 0 values against the indexing based fields instead of omitting them completely.